.github/workflows/merlin.yml

```diff
@@ -137,7 +137,9 @@ jobs:
         run: make unit-test
 
   lint-api:
-    runs-on: ubuntu-latest
+    # Downgrade the runner image to ubuntu-20.04 temporarily as it's failing sporadically;
+    # see https://github.com/actions/runner-images/issues/6709 for more details
+    runs-on: ubuntu-20.04
```

Comment:
Somehow this isn't working, all this did was to reveal the GitHub runner running the lint job OOM-ed. I'm still trying to find a way to fix this.